tests/lib/rules/indent.js

```diff
@@ -289,7 +291,20 @@ eslintTester.addRuleTest("lib/rules/indent", {
                 "        break;\n" +
                 "}",
             options: [4, {indentSwitchCase: true}],
-            errors: expectedErrors([[13, 4], [15, 4], [17, 4]])
+            errors: expectedErrors([[5, 8], [8, 8]])
+        },
+        {
+            code:
+                "switch (a) {\n" +
+                "case '1':\n" +
+                "b();\n" +
+                "break;\n" +
+                "default:\n" +
+                "c();\n" +
+                "break;\n" +
+                "}",
+            options: [4, {indentSwitchCase: true}],
+            errors: expectedErrors([[3, 4], [4, 4], [6, 4], [7, 4]])
```

Comment:
With `indentSwitchCase` false, case should align with switch while all statements in the case should be indented one level. When it's true, case should be indented one level from switch and the statements in the case should be indented one level from the case. Break has no special treatment and should be indented along with everything else in the case.